assets/src/edit-story/components/panels/videoAccessibility.js

```diff
@@ -50,6 +50,7 @@ function VideoAccessibilityPanel({ selectedElements, pushUpdate }) {
       pushUpdate(
         {
           poster: image.sizes?.medium?.url || image.url,
+          posterGenerated: false,
```

Comment:
Why is this here? The `posterGenerated` should only belong in the `Resource` object for videos. It has no bearing on the element itself.

Reply:
If you update the poster, it is not longer a generated poster and the height and size can not be trusted.